homeassistant/components/tuya/light.py

```diff
@@ -210,14 +220,10 @@ class TuyaLightEntityDescription(LightEntityDescription):
     "sp": (
         TuyaLightEntityDescription(
             key=DPCode.FLOODLIGHT_SWITCH,
+            icon="mdi:light-flood-down",
             brightness=DPCode.FLOODLIGHT_LIGHTNESS,
             name="Floodlight",
         ),
-        TuyaLightEntityDescription(
-            key=DPCode.BASIC_INDICATOR,
-            name="Indicator Light",
-            entity_category=EntityCategory.CONFIG,
-        ),
```

Comment:
Not sure why this is remove? This is a light and should thus be part of the light domain.